Repository for shiny app to explore data associated with paper "Neuronal Activity-Dependent Gene Dysregulation in C9orf72 i3Neuronal Models of ALS/FTD Pathogenesis"

Abstract
The GGGGCC nucleotide repeat expansion (NRE) mutation in the C9orf72 (C9) gene is the most common cause of ALS and FTD. Neuronal activity plays an essential role in shaping biological processes within both healthy and neurodegenerative disease scenarios. Here, we show that at baseline conditions, C9-NRE iPSC-cortical neurons display aberrations in several pathways, including synaptic signaling and transcriptional machinery, potentially priming diseased neurons for an altered response to neuronal stimulation. Indeed, exposure to two pathophysiologically relevant stimulation modes, prolonged membrane depolarization, or a blockade of K+ channels, followed by RNA sequencing, induces a temporally divergent activity-dependent transcriptome of C9-NRE cortical neurons compared to healthy controls. These findings reveal the impact of neuronal activity on the ALS/FTD-associated transcriptome and may reveal pathways necessary for conferring neuronal resilience or degeneration.


Comparative gene expression analysis of RNAseq data for human i3Neurons from the following conditions: Untreated (UT), TTX-silenced for 16 hours or for 24 hours(TTX), potassium chloride(KCL), or tetraethylammonium (TEA). Two or three independent differentiations for 3 different i3Neurons lines were used for each condition.


To launch shiny app in R, use shiny(v1.9.1) and the command:
  runGitHub("LTG_shiny", "arhaeusler")

If using, please cite paper:
(citation here)

Publicly available RNA-seq data found on NCBI GEO (GEO accession # here)
